Library/Formula/bazel.rb

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
Please don't add commented out lines that aren't relevant to the formulae itself.